### ISSUE TITLE
Clinician: access to a team inbox

### ISSUE THREAD

**Issue by MollieCourtenay** [healthlocker/healthlocker#325]
As a clinician, I am part of a team.
When I launch Healthlocker from ePJS, I am presented with the option to open a team inbox.
This is where all unread messages from Healthlocker users (service users and carers) will be listed.
This is to ensure if a team member is off work, there is still visibility of incoming messages from Healthlocker users.

______
I can respond to a message from within the team mailbox section in Healthlocker.
I can also choose to click on the users name to launch their Healthlocker page. This is so I can read previous messages or gain some insight or context before replying.

**Comment by katbow** [comment]
I'm going to hold off on a time estimate for this one until there's a discussion about it. Will add when there is more info.

**Comment by iteles** [comment]
Agreed we would hold off this feature until it comes back as a request from the actual users.
In the meantime, all clinicians who are assigned to the user will be able to interact with them over messages.
